packages/render-fidelity-tools/src/workflows/test-fidelity.ts

```diff
@@ -22,17 +22,19 @@ const require = module.createRequire(import.meta.url);
 
 import {ArtifactCreator} from '../artifact-creator.js';
 
+
 const configPath = resolve(process.argv[2]);
 const rootDirectory = resolve(dirname(configPath));
 const config = require(configPath);
 
 const outputDirectory = join(rootDirectory, 'results');
+const portal = 9030;
```

Comment:
Were these changes intended to be committed?

Reply:
Yeah, do you think it's a good way to refactor code?

Reply:
maybe just call it port; that's more commonly used.